src/main.ts

```diff
@@ -458,9 +458,12 @@ const checkNvidiaSmi = async () => {
                 );
 
                 vramChecker.stdout.on('data', (data) => {
-                    const [, vramTotal, vramUsed] = String(data).split('\n')[0].split(', ');
+                    const [, vramTotal, vramUsed] = String(data).split(/\s*,\s*/, 4);
```

Comment:
Does split not require the regex multiline flag to handle multiline strings?

Reply:
The multiline flag (m) only changes the meaning of `^`and `$`.

Reply:
Good to know. Tbh, never used that flag before, but just wanted to check.